src/Judopay/Model/TokenPayment.php

```diff
@@ -19,7 +19,8 @@ class TokenPayment extends \Judopay\Model
         'cv2' => DataType::TYPE_STRING,
         'consumerLocation' => DataType::TYPE_ARRAY,
         'mobileNumber' => DataType::TYPE_STRING,
-        'emailAddress' => DataType::TYPE_STRING
+        'emailAddress' => DataType::TYPE_STRING,
+        'clientDetails' => DataType::TYPE_ARRAY,
```

Comment:
Ok.  When testing and looking at the logs, it looked like this worked, but will leave it in your hands ;)